sos/plugins/pmem.py

```diff
@@ -0,0 +1,77 @@
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+
+from sos.plugins import Plugin, RedHatPlugin, UbuntuPlugin, DebianPlugin
+
+
+class Pmem(Plugin, RedHatPlugin, UbuntuPlugin, DebianPlugin):
+    """Persistent Memory devices
+    """
+
+    plugin_name = 'pmem'
+    profiles = ('storage', 'hardware', 'memory')
+
```

Comment:
You'll need to add some package or file triggers here: since you are enabling the plugin for all supported distro policies you'll need to know the corresponding package names, or well-known-paths (this might involve splitting the plugin into different sub-classes, like `RedHatPmem`, `DebianPmem` if those packages or paths differ).

Reply:
The tools are generic across most distros but not yet available in their package repo's, so I added both the package and common search paths.